src/stories/components/TransparencyCard/TransparencyCard.tsx

```diff
@@ -19,24 +19,27 @@ export const TransparencyCard = (props: Props) => {
     <Container isLight={isLight}>
       <HeaderWrapper>{props.header}</HeaderWrapper>
       {props.headers.map((header, i) => (
-        <Row
-          key={header.toString()}
-          hasIcon={header !== 'Target Balance' || (header === 'Target Balance' && props.itemType === 'total')}
-        >
-          <Label hasIcon={header === 'Target Balance'}>{header}</Label>
-          <div
-            style={{
-              display: props.itemType === 'total' ? 'flex' : undefined,
-              justifyContent: props.itemType ? 'flex-end' : undefined,
-              width:
-                header === 'Target Balance' || (header === 'Target Balance' && props.itemType !== 'total')
-                  ? '100%'
-                  : undefined,
-            }}
+        <>
+          <Row
+            key={header.toString()}
+            hasIcon={header !== 'Target Balance' || (header === 'Target Balance' && props.itemType === 'total')}
```

Comment:
This looks kinda a patch, and in the future, more icons will be added to other columns, or the Column names could change using this approach will make really hard to change the names of the columns and keep it bugs free. Could you use any different approach to archive this?